package.json

```diff
@@ -13,5 +13,6 @@
     "pre-commit:ui-react": "cd app/ui-react && yarn pre-commit",
     "pre-push:ui-react": "./tools/bin/syndesis -m ui-react --incremental",
     "postinstall": "husky install"
-  }
+  },
+  "version": "0.0.0"
```

Comment:
Should this be included? Looks a little odd.

Reply:
Not sure about this one too, I will remove it.